docs/modules/theme/pages/fallback-font.adoc

```diff
@@ -95,3 +95,28 @@ font:
 ----
 
 Now Asciidoctor PDF will look for the emoji in the Symbola font instead of the Noto Emoji font.
+
+== Default Microsoft Windows Font Limitations
+When using the fonts provided by default in Microsoft Windows 10, and possibly other versions, it is important to define a fallback font to provide full coverage of Asciidoctor language features.
+In particular, the https://docs.asciidoctor.org/asciidoc/latest/lists/checklist/[Checklist] feature makes use of the _Ballet Box_ ☐ (U+2610) and _Ballot Box with Check_ "☑" (U+2611) characters.
+Two fonts provided by Windows, Segoe UI Symbol (`seguisym.ttf`) and Segoe UI Emoji (`seguiemj.ttf`), do contain these characters.
+These two fonts have baselines which match better with most of the Windows font library than the default Notoserif font which is included with Asciidoctor PDF.
+
+Here is an example using Times New Roman with Segoe UI Symbol as a fallback providing the characters needed to support checklists:
+[,yaml]
+----
+font:
+  catalog:
+    TimesNewRoman:
+      normal: times.ttf
+      italic: timesi.ttf
+      bold: timesbd.ttf
+      bold_italic: timesbi.ttf
+    SegoeUISymbolFallback: seguisym.ttf
+  fallbacks:
+  - SegoeUISymbolFallback
+----
+
+Make sure to configure the theme to include the location of the Windows font files, which is `c:\Windows\Fonts`.
```

Comment:
When referencing paths, please use the following syntax:

```
[.path]_c:/Windows/Fonts_
```

Always use forward slashes in Windows file paths (as it's both legal and avoids backslash escaping issues).

Reply:
Implemented in commit 48c098682b30b0afd6feafaa52614244d4f76c0f.